components/event-processor/org.wso2.carbon.event.processor.ui/src/main/resources/web/eventprocessor/js/create_execution_plan_helper.js

```diff
@@ -180,17 +181,33 @@ function addImportedStreamDefinition() {
                     }
                 }
             }
-            var importStatement = ANNOTATION_TOKEN_AT +
-                ANNOTATION_IMPORT +
-                ANNOTATION_TOKEN_OPENING_BRACKET +
-                SIDDHI_SINGLE_QUOTE + streamId + SIDDHI_SINGLE_QUOTE +
-                ANNOTATION_TOKEN_CLOSING_BRACKET +
-                SIDDHI_LINE_BREAK +
-                SIDDHI_LITERAL_DEFINE_STREAM + SIDDHI_SPACE_LITERAL + streamAs + SIDDHI_SPACE_LITERAL +
-                ANNOTATION_TOKEN_OPENING_BRACKET +
-                streamDefinition +
-                ANNOTATION_TOKEN_CLOSING_BRACKET +
-                SIDDHI_STATEMENT_DELIMETER;
+            var importStatement;
+            if(propImportArbitrary){
```

Comment:
Hi Thilini,

This is fixed by the commits in : https://github.com/wso2/carbon-event-processing/pull/266

Thanks,
SajithD